services/wallet/datastore.go

```diff
@@ -63,7 +64,7 @@ func init() {
 // Datastore holds the interface for the wallet datastore
 type Datastore interface {
 	datastore.Datastore
-	LinkWallet(ctx context.Context, id string, providerID string, providerLinkingID uuid.UUID, depositProvider, country string) error
+	LinkWallet(ctx context.Context, id string, providerID string, providerLinkingID uuid.UUID, depositProvider string) error
```

Comment:
```suggestion
	LinkWallet(ctx context.Context, id, providerID string, providerLinkingID uuid.UUID, depositProvider string) error
```

Reply:
This is the legacy signature, I am going to leave this as it will be removed/changed to `UUID` in the next PR, plus this will change will also require rerunning the instrumented datastore again... Also see [comment](https://github.com/brave-intl/bat-go/pull/2280/files#r1446183200)